src/redux/books/books.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+
+const Books = () => (
+  <div className="book-list">
+    <ul>
+      <li>
+        <span className="title">The Hunger Games</span>
+        <button type="button">Remove</button>
+      </li>
+      <li>
+        <span className="title">Dune</span>
+        <button type="button">Remove</button>
+      </li>
+      <li>
+        <span className="title">Capital in the Twenty-First Century</span>
+        <button type="button">Remove</button>
+      </li>
+    </ul>
+
+    <h2>Add a book</h2>
+    <form>
+      <input type="text" placeholder="Add a Book" />
+      <button type="button">Add</button>
+    </form>
```

Comment:
- [ ] Your form is missing the field for a book author. Kindly add it.
- [ ] Kindly extract this form into a new component and render that component here instead.